defaults/main.yml

```diff
@@ -1,14 +1,23 @@
 ---
 varnish_package_name: "varnish"
 varnish_version: "4.1"
+# Varnish 5+ is currently only supported on Debian and Ubuntu.
+# varnish_version: "5.1"
 
 varnish_use_default_vcl: true
 varnish_default_vcl_template_path: default.vcl.j2
 
 varnish_default_backend_host: "127.0.0.1"
 varnish_default_backend_port: "8080"
+# If backend server expect PROXY protocol connexion:
+# varnish_default_backend_proxy_proto: 1
+
+varnish_listen_ports:
+  - port: 80
+  - port: 6081
+    # If Varnish must expect a PROXY protocol connexion:
+    proxy_proto: yes
```

Comment:
If you can merge commits one by one, then first merge commits ca8fae7 and 16a8b43 (which basically only fixes minor issues), and then the d3465ff commit contains all Varnish 5 related changes.

And after that, you could look at c85b1dc and e2319ff, which are only related to PROXY support.

Is it possible?